src/desktop/apps/partner2/client/view.coffee

```diff
@@ -15,6 +15,7 @@ ContactView = require './contact.coffee'
 tablistTemplate = -> require('../templates/tablist.jade') arguments...
 mediator = require '../../../lib/mediator.coffee'
 { Following, FollowButton } = require '../../../components/follow_button/index.coffee'
+{ ContextModule } = require "@artsy/reaction/dist/Artsy/Analytics/v2/Schema"
```

Comment:
Looking over this PR and seeing how many use-cases we have for importing directly from a sub namespace, it makes sense. So revising my other comment, only concerned about avoiding `* as` (not actionable now of course)